docs/enterprise-setup/implementation-guide.md

```diff
@@ -393,23 +426,56 @@ Once this is complete, ensure that the value of the `webapp-url` field in your `
 
 You may configure ingress using a load balancer or an API Gateway. We do not currently support most service meshes (such as Istio). If you are having networking issues after fully deploying Airbyte, please verify that firewalls or lacking permissions are not interfering with pod-pod communication. Please also verify that deployed pods have the right permissions to make requests to your external database.
 
-### Install Airbyte Enterprise
+### Step 4: Deploy Self-Managed Enterprise
 
-Install Airbyte Enterprise on helm using the following command:
+Install Airbyte Self-Managed Enterprise on helm using the following command:
 
 ```sh
-./tools/bin/install_airbyte_pro_on_helm.sh
+helm upgrade \
```

Comment:
as this is the _install_ step, we should probably use `helm install` instead of `helm upgrade --install`

Reply:
fixed